modules/swagger-codegen/src/main/resources/JavaSpring/libraries/spring-boot/pom.mustache

```diff
@@ -44,6 +44,10 @@
             <artifactId>spring-boot-starter-tomcat</artifactId>
             <scope>provided</scope>
         </dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
```

Comment:
ok @cbornet 
Next action :
>> Change dependency to spring-data-common
>> SpringFox plugin parameter
But I have to leave , I could not go back to the code before mid-August